internal/backend/remote-state/consul/client.go

```diff
@@ -181,7 +181,7 @@ func (c *RemoteClient) Put(data []byte) error {
 			// the user. We may end up with dangling chunks but there is no way
 			// to be sure we won't.
 			path := strings.TrimRight(c.Path, "/") + fmt.Sprintf("/tfstate.%s/", hash)
-			kv.DeleteTree(path, nil)
+			kv.DeleteTree(path, nil) //nolint:errcheck
```

Comment:
```suggestion
			_, _ = kv.DeleteTree(path, nil)
```
Nit: In cases where it's easy to write (As opposed to ignoring errors when starting a  go-routine) and we agree that ignoring the error is a safe option, I think we prefer to use a simple Go syntax instead of *nolint* rules